lib/screens/search_screen.dart

```diff
@@ -70,44 +72,210 @@ class _SearchScreenState extends State<SearchScreen> {
 
   String? _error;
 
-  /// List of movies matching the search query.
+  /// Comprehensive search results categorized by search type.
```

Comment:
Does this private class ever get documented? If not we should not be using docstring comments `///`.  I ran `dart doc` to check and it seems not.